pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -1303,3 +1313,57 @@ func CleanupLegacyLokiPriorityClass(ctx context.Context, seedClient client.Clien
 
 	return client.IgnoreNotFound(seedClient.Delete(ctx, priorityClass))
 }
+
+// CleanupOldFluentBit deletes all old fluent-bit resources which are not installed by the fuent-operator
+func CleanupOldFluentBit(ctx context.Context, seedClient client.Client) error {
+	// Resources which does not duplicate these from the operators
+	uniqueResource := []client.Object{
+		&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit-config", Namespace: v1beta1constants.GardenNamespace}},
+		&schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit"}},
```

Comment:
Hm, the `fluent-bit` PriorityClass should be already cleaned up. I dropped the legacy deletion code in https://github.com/gardener/gardener/pull/7433/files. Hence, no need to delete it here:
```suggestion
```